modules/client/examples.md

```diff
@@ -1392,7 +1411,6 @@ for await (const step of steps) {
         console.log(step.txHash);
         break;
       case VoteProposalStep.DONE:
-        console.log(step.voteId);
```

Comment:
Removing this line?

Reply:
Yes, this returned the `proposalId` for some reason and didn't make much sense. In the contracst there is no `voteId` so I removed it